### PR TITLE
Require OpenWrt for setup.sh and uninstall.sh

### PR DESCRIPTION
**Pull request by rany2**

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -13,6 +13,15 @@ main() {
 	# Set correctness options
 	set -eu
 
+	# Check if OS is OpenWRT
+	unset ID
+	. /etc/os-release 2>/dev/null || true
+	if [ "${ID:-}" != "openwrt" ]
+	then
+		printf "This script requires OpenWRT.\n"
```
Comment (lynxthecat) [inline setup.sh:21]:
```suggestion
		printf "This script requires OpenWrt.\n"
```

File: uninstall.sh [hunk 2]
```diff
@@ -13,6 +13,15 @@ main() {
 	# Set correctness options
 	set -eu
 
+	# Check if OS is OpenWRT
+	unset ID
+	. /etc/os-release 2>/dev/null || true
+	if [ "${ID:-}" != "openwrt" ]
+	then
+		printf "This script requires OpenWRT.\n"
```
Comment (lynxthecat) [inline uninstall.sh:21]:
```suggestion
		printf "This script requires OpenWrt.\n"
```

**Comment by lynxthecat** [comment]
Isn't it mean to be 'OpenWrt': https://openwrt.org/start

image

**Comment by rany2** [comment]
> Isn't it mean to be 'OpenWrt': https://openwrt.org/start
> 
> image

Why would it be mean?

**Comment by lynxthecat** [comment]
Sorry I meant 'meant' rather than 'mean'!

**Pull request merged**